core/drivers/d06_lpc_uart.c

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2021, HiSilicon Limited
+ */
+
+#include <assert.h>
+#include <drivers/d06_lpc_uart.h>
+#include <io.h>
+#include <keep.h>
+#include <mm/core_mmu.h>
+#include <util.h>
+
+static paddr_t lpc_base = LPC_BASE;
```

Comment:
So this driver cannot be used for multiple devices? The code would be much cleaner without this restriction.

Reply:
fixed. thx